### DISCUSSION TITLE
The question here is can / will i be able to use prisma directly wit rust, using the binary.

### DISCUSSION THREAD

**Discussion by crazyaccess** [prisma/prisma#6939] [Q&A] [answered]
I am using prisma with node for a lot of time now, and I was thinking whether there there is already a way of using prisma directly with rust ???

**Comment by pantharshit00** [comment, marked as answer]
There is no official rust client for Prisma yet. Please follow https://github.com/prisma/prisma/issues/5790 for that. 

But you can checkout our SQL abstraction library for Rust for now which we also use inside of Prisma: https://github.com/prisma/quaint/
